tests/provision_basic_win.ps1

```diff
@@ -0,0 +1,12 @@
+# Variables
+$log = "C:/vagrant/puppet-agent.log"
+$agent_url = "https://s3.amazonaws.com/puppet-agents/2017.2/puppet-agent/1.10.4/repos/windows/puppet-agent-1.10.4-x64.msi"
+if ( Get-Command "puppet" -ErrorAction SilentlyContinue ) {
+  Write-Output "Puppet is already installed.  Skipping install of $agent_url"
+} else {
+  Write-Output "Installing Puppet from $agent_url"
+  Write-Output "Log will be written to $log"
+  Remove-Item $log
```

Comment:
why the remove here? seems to be causing issues.

```
==> win2012r2-client: Log will be written to C:/vagrant/puppet-agent.log
==> win2012r2-client: Remove-Item : Cannot find path 'C:\vagrant\puppet-agent.log' because it does
==> win2012r2-client: not exist.
==> win2012r2-client: At C:\tmp\vagrant-shell.ps1:9 char:3
==> win2012r2-client: +   Remove-Item $log
==> win2012r2-client: +   ~~~~~~~~~~~~~~~~
==> win2012r2-client:     + CategoryInfo          : ObjectNotFound: (C:\vagrant\puppet-agent.log:Str
==> win2012r2-client:    ing) [Remove-Item], ItemNotFoundException
==> win2012r2-client:     + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.Remov
==> win2012r2-client:    eItemCommand
```